data/src/tensor/indices.rs

```diff
@@ -0,0 +1,73 @@
+pub struct IndexIterator {
```

Comment:
is this better / different than ndarray::indexes() ?

Reply:
After re-checking, `ndarray::indexes()` does the job !